docs/content/docs/dev/configuration/gradle.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Using Gradle"
+weight: 3
+type: docs
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# How to use Gradle to configure your project
+
+You will likely need a build tool to configure your Flink project. This guide will show you how to
+do so with [Gradle](https://gradle.org), an open-source general-purpose build tool that can be used 
+to automate tasks in the development process.
+
+## Requirements
+
+- Gradle 7.x 
+- Java 8.x
+
+## Importing the project into your IDE
+
+Once the project folder and files have been created, we recommend that you import this project into
+your IDE for developing and testing.
```

Comment:
Which project folder and files? Which project should be imported?
Are we missing a reference to the gradle quickstart?